src/modules/temperature.cpp

```diff
@@ -44,10 +46,15 @@ namespace modules {
   bool temperature_module::update() {
     m_temp = std::atoi(file_util::contents(m_path).c_str()) / 1000.0f + 0.5f;
     m_perc = math_util::cap(math_util::percentage(m_temp, 0, m_tempwarn), 0, 100);
+    int temp_f = floor((((9.0/5.0) * m_temp) + 32) + 0.5);
```

Comment:
Why the `+0.5` here?

Reply:
I think it's for rounding purposes. Usually `floor()` isn't used after that though, because the value is already rounded.